src/modules/common/Config.ts

```diff
@@ -545,4 +608,29 @@ export interface IConfig {
      * Consensus config
      */
     consensus: IConsensusConfig;
+    /**
+     * Votera config
+     */
+    votera: IVoteraConfig;
+
+}
+
+/**
+ * The interface of Votera 
+ */
+export interface IVoteraConfig {
+    /**
+     * THe address to which we bind
+     */
+    address: string;
+
+    /**
+     * The port on which we bind
+     */
+    port: number;
+
+    /**
+     * The endpoint of votera
+     */
+    votera_endpoint: URL;
```

Comment:
You only need one votera endpoint or an address and port.

Reply:
yes this is write but i want to keep the things similar. this will help in code extenstion if there is any change.